prov/verbs/configure.m4

```diff
@@ -44,6 +44,17 @@ AC_DEFUN([FI_VERBS_CONFIGURE],[
 				[verbs_rdmacm_happy=0])
 	      ])
 
+	AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
+			   [
+			    #include <infiniband/verbs_exp.h>
+			   ],
+			   [
+			    return (IBV_EXP_DEVICE_ATTR_ODP | IBV_EXP_DEVICE_ODP);
```

Comment:
I want just to check that IBV_EXP_DEVICE_ATTR_ODP and IBV_EXP_DEVICE_ODP defined, i.e. I tried `#if defined(IBV_EXP_DEVICE_ATTR_ODP) && defined(IBV_EXP_DEVICE_ODP)`, but it is unsuccessful for old and new versions of OFED. I looked into `"infiniband/verbs_exp.h"`  and saw that those are items of enum.
My current solution is if specified item of enum is defined, the code should be compiled, due to this I just try to use `IBV_EXP_DEVICE_ATTR_ODP`, `IBV_EXP_DEVICE_ODP` in `configure`. And this works :)